DigitalLearningSolutions.Web/Controllers/LearningMenuController/LearningMenuController.cs

```diff
@@ -79,6 +81,22 @@ public IActionResult Section(int customisationId, int sectionId)
             return View("Section/Section");
         }
 
+        [Route("/LearningMenu/{customisationId:int}/{sectionId:int}/{tutorialId:int}")]
+        public IActionResult Tutorial(int customisationId, int sectionId, int tutorialId)
+        {
+            var candidateId = User.GetCandidateId();
+            sessionService.StartOrUpdateSession(candidateId, customisationId, HttpContext.Session);
+
+            var tutorialContent =
+                tutorialContentService.GetTutorialContent(candidateId, customisationId, sectionId, tutorialId);
+
+            if (tutorialContent == null)
+            {
+                return RedirectToAction("StatusCode", "LearningSolutions", new { code = 404 });
+            }

```

Comment:
You also need to call the method to update the progress 🙂

Reply:
Yep! Done in 825a307